doc/_static/css/custom-theme.css

```diff
@@ -154,6 +158,31 @@ pre> :not(:first-child).ipynb-lexed * {
     border-right: 2px solid #eee;
 }
 
+.mobile {
+    display: none;
+}
+
+.horizontal-menu {
+    width: 100%;
+    margin-top: 7px;
+}
+
+.horizontal-menu ul {
+    display: grid;
+    grid-template-columns: auto auto auto auto auto;
+    width: inherit;
+}
+
+.horizontal-menu a.nav-link {
+    color: #fff;
+    padding: 5px;
+    font-size: 14px;
+}
+
+.horizontal-menu .nav-item.active {
+    border-bottom: 3px solid #fff;
```

Comment:
Some of the stuff here are px, won't it affect responsiveness?

Reply:
In order to keep it consistent I changed font-sizes to rem where 1rem=16px, but since some stuff has already been written in px and some in rem, it won't affect responsiveness at this point.